lib/Installation/ModuleRegistration/ModuleHARegCodePage.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The module provides interface to act with page that ask for module extra registration code
+#
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::ModuleRegistration::ModuleHARegCodePage;
```

Comment:
I bet we can reuse this page for other regcodes, we just need to list them there when we need them. so we can call it `ModuleRegCodePage`. method `set_ha_regcode` is fine and there will be more specific methods for other products which require additional regcodes. wdyt?